### ISSUE TITLE
Bug: HTTP_CONTROL_SERVER_PORT doesn't control healthcheck

### ISSUE THREAD

**Issue by stumpylog** [qdm12/gluetun#272]
**TLDR**: When environment variable HTTP_CONTROL_SERVER_PORT is set, the container reports as unhealthy.

I'll just change it back to the normal and re-direct in my compose file.  But maybe there's a way to fix it in code too.  Or the documentation should mention this?

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What VPN service provider are you using?

    - [x] PIA
    - [ ] Mullvad
    - [ ] Windscribe
    - [ ] Surfshark
    - [ ] Cyberghost

3. What's the version of the program?
`Running version latest built on 2020-10-26T02:52:47Z (commit 0c81154)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

```log
2020-10-26T11:19:20.881-0700	INFO	http server: listening on 0.0.0.0:8888
```

Status | unhealthy
-- | --
Failure count | 6
Last output | Get "http://localhost:8000/health": dial tcp 127.0.0.1:8000: connect: connection refused

Host OS: Ubuntu 20.04

**Comment by qdm12** [comment]
Ah indeed, a bug! Thanks for reporting!
